draft-ietf-ipsecme-ikev2-sa-ts-payloads-opt.md

```diff
@@ -202,22 +200,21 @@ The OPTIMIZED_REKEY Notify Message type is used to perform an optimized IKE SA o
 +---------------+-+-------------+-------------------------------+
 | Next Payload  |C|  RESERVED   |         Payload Length        |
 +---------------+-+-------------+-------------------------------+
-|Protocol ID    | SPI Size (=8) |      Notify Message Type      |
+|Protocol ID(=0)| SPI Size (=0) |      Notify Message Type      |
 +---------------+---------------+-------------------------------+
-|                Security Parameter Index (SPI)                 |
+|                                                               |
+~                            New SPI                            ~
 |                                                               |
 +---------------------------------------------------------------+
 ~~~~
 
-* Protocol ID (1 octet) - For an IKE SA rekey, this field MUST contain (1). For Child SAs, this field MUST contain either (2) to indicate AH or (3) to indicate ESP.
+* Protocol ID (1 octet) - MUST be 0.
 
-* SPI Size (1 octet) - MUST be 8 when rekeying an IKE SA. MUST be 4 when rekeying a Child SA.
+* SPI Size (1 octet) - MUST be 0, meaning no SPI is present.

```

Comment:
Perhaps this should be made clearer. Readers may be confused how "no SPI is present" is consistent with "New SPI". Perhaps a clarification text should be added that the "Security Parameter Index (SPI)" field in the Notify payload is not used and the new SPI is placed into the "Notification Data" field.

Reply:
Done the updates in the new version.